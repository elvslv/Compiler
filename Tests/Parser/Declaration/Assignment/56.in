type trec = record x, y: integer; end;
var a, b: trec;
begin
	a := b + a;
end.