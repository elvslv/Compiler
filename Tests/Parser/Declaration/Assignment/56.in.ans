type TREC = record 
    var X : INTEGER;
    var Y : INTEGER;
end;
var A : TREC;
var B : TREC;
10 4 Invalid types in binary operation