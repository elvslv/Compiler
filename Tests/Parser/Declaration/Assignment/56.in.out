type TREC = record 
    var X : INTEGER;
    var Y : INTEGER;
end;
var A : TREC;
var B : TREC;
15 4 Invalid types in binary operation